lib/http_proxy.c

```diff
@@ -169,7 +169,7 @@ static CURLcode dynhds_add_custom(struct Curl_easy *data,
                  Connection: */
               hd_name_eq(name, namelen, STRCONST("Connection:")))
         ;
-      else if((conn->httpversion >= 20) &&
+      else if((httpversion >= 20) &&
               hd_name_eq(name, namelen, STRCONST("Transfer-Encoding:")))
         /* HTTP/2 does not support chunked requests */
```

Comment:
Maybe adjust this comment while at it? Should say **HTTP/2 and HTTP/3**, shouldn't it?

Reply:
Done.